stable/artifactory/values.yaml

```diff
@@ -175,6 +179,7 @@ artifactory:
   ## ARTIFACTORY_HOME/etc/artifactory.lic and loaded at run time.
   ## The dataKey should be the name of the secret data key created.
   license:
+    licenseKey:
```

Comment:
Add a note that only `licenseKey` or `secret` should be used or that `secret` will override `licenseKey` if both are passed.

Reply:
@eldada Done